ci/pre-2node.yml

```diff
@@ -0,0 +1,34 @@
+---
+- name: "Do pre-work to get kubeconfig"
+  hosts: controller
+  vars:
+    ci_framework_dir: "{{ ansible_user_dir }}/{{ zuul.projects['github.com/openstack-k8s-operators/ci-framework'].src_dir }}"
+  environment:
+    PATH: "~/.crc/bin:~/.crc/bin/oc:~/bin:{{ ansible_env.PATH }}"
+  tasks:
+      # this task is temperory, eventually, this whole play will be in a separate file, which will do the common tasks already.
+    - name: "Set the sto_dir if it isn't already set"
+      ansible.builtin.set_fact:
+        sto_dir: '{{ ansible_env.HOME }}/{{ zuul.projects["github.com/infrawatch/service-telemetry-operator"].src_dir }}'
+      when: sto_dir | default('') | length == 0
+
+    - name: "Run bootstrap playbook"
+      ansible.builtin.shell:
+        cmd: |
+          ansible-playbook -e@{{ ansible_user_dir }}/ci-framework-data/artifacts/parameters/zuul-params.yml {{ ci_framework_dir }}/ci_framework/playbooks/01-bootstrap.yml
```

Comment:
From https://github.com/openstack-k8s-operators/ci-framework/pull/943, the ci-framework has moved to an ansible collection and the playbooks are in the repo root, instead of inside `ci_framework`. For now the path as written here will still work since we have a symlink, but if this PR needs to be updated it would be good to remove the `ci_framework` from the path. Note that it will work as is, so it could be done in a follow-up without any issue.

Reply:
Thanks @cescgina, I'll update this PR to do that. I have a few more pending updates for the job as well, and can roll these in.